app/models/Integration.ts

```diff
@@ -29,6 +30,45 @@ class Integration<T = unknown> extends Model {
 
   @observable
   settings: IntegrationSettings<T>;
+
+  isMentionable(url: URL) {
```

Comment:
Not blocking – I'm not certain these helpers belong here, could be some sort of mention lib / `MentionHelper` would be better.